es6-lib/upstream/zookeeper.js

```diff
@@ -48,15 +48,12 @@ class Zookeeper extends EventEmitter {
 
   _getCore(cb) {
     this._client.getChildren(CORE_PATH, (err, children) => {
-      if (err) return cb(err);
-      if (!children.length) return cb({
-        statusCode: 502,
-        body: 'No core nodes registered in zookeeper'
-      });
+      if (err) return cb(new ZKError('Failed to get zookeeper children'));
+      if (!children.length) return cb(new ZKError('No core nodes registered in zookeeper'));
       var chosen = children[Math.floor(Math.random() * children.length)];
       var instance = path.join(CORE_PATH, chosen);
       this._client.getData(instance, (err, buf) => {
-        if (err) return cb(err);
+        if (err) return cb(new ZKError('Error getting data from zookeeper'));
```

Comment:
Can there be curly braces around this `if` and the ones above?  The `return` gets kind of lost when it's a one-liner.